client/components/text/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { PropTypes } from 'react';
+import FormFieldset from 'components/forms/form-fieldset';
+
+const Text = ( { id, layout, value } ) => {
+	console.log( layout );
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/398/commits/1e76910535dc441ee89832ea26ffd36057bd6fff